testing/rclone/APKBUILD

```
@@ -0,0 +1,98 @@
+# Contributor: tcely <rclone+aports@tcely.33mail.com>
+# Maintainer: tcely <rclone+aports@tcely.33mail.com>
+_pkgname="rclone"
+pkgname="${_pkgname}"
+pkgver="1.48.0"
+pkgrel=0
+pkgdesc="rsync for cloud storage"
+url="https://rclone.org/"
+arch="all"
+license="MIT"
+options=""
+depends=""
+makedepends="bash git go perl"
+install=""
+subpackages=""
+
+gpg_signature_extensions="."
+gpgfingerprints="good:FBF7 37EC E9F8 AB18 604B  D2AC 9393 5E02 FF3B 54FA"
+gpgsource="${pkgname}-${pkgver}-sha256sums::https://downloads.rclone.org/v${pkgver}/SHA256SUMS"
+source="https://downloads.rclone.org/v${pkgver}/${_pkgname}-v${pkgver}.tar.gz $gpgsource"
+
+builddir="${srcdir}/${_pkgname}-v${pkgver}"
+export GOPATH="${srcdir}/go"
+
+sanitycheck() {
+	local source= sha512sums=
+	default_sanitycheck
+}
+
+cleanup_srcdir() {
+	go clean -modcache
+	default_cleanup_srcdir
+}
+
+build() {
+	make rclone
+}
+
+check() {
+	make quicktest
```

The Makefile for rclone is really weird, and might become incompatible with GO111MODULES later, which are used.
We also want to run more tests, imo.
```suggestion
	GO111MODULE=on go test ./...
```